vm/vm.go

```diff
@@ -50,6 +50,7 @@ type callContext struct {
 	response []*felt.Felt
 	// amount of gas consumed per transaction during VM execution
 	gasConsumed []*felt.Felt
+	traces      []string
```

Comment:
```suggestion
	traces      []json.RawMessage
```

so we dont need to convert it when returning from `vm.Execute`?


``` golang
	var traces []json.RawMessage
	for _, trace := range context.traces {
		traces = append(traces, json.RawMessage(trace))
	}
```